src/modules/organizations/containers/projects-container.jsx

```diff
@@ -38,31 +39,36 @@ class ProjectsContainer extends React.Component {
   }
 
   getLinkedProjects(organization = this.props.organization, page = 1) {
-    if (organization) {
+    if (organization && organization.links.projects) {
       const query = { sort: 'display_name', page };
 
-      organization.get('projects', query).then((projects) => {
-        const allProjects = organization.links.projects.map((projectId) => {
-          const proj = projects.find(project => project.id === projectId);
-          if (proj) {
-            return proj;
-          }
-          return {
-            display_name: `Project ${projectId}`,
-            id: projectId,
-            links: {
-              owner: {
-                display_name: 'CHECK WITH OTHER ORG COLLABORATORS'
+      const projectIds = organization.links.projects;
```

Comment:
If it's alright with you, can we leave this comment above this variable?
`// Isolate the project Ids associated with the organization`

The fact that `organization.links.projects` returns an array of project ids is not particularly clear from the project variable's name (comment wouldn't feel as necessary if it were, eg, `organization.links.projectIds`), so it seems like an appropriate place to leave a comment due to the naming choice.

Or if you prefer alternate wording, feel free to edit the comment, but I think it's a good place to make clear what the variable is doing, since the naming is a little misleading.